OpenQA/Isotovideo/Utils.pm

```diff
@@ -12,15 +12,17 @@ use bmwqemu;
 use autotest;
 use Try::Tiny;
 
-our @EXPORT_OK = qw(checkout_git_repo_and_branch checkout_git_refspec
-  handle_generated_assets load_test_schedule);
+our @EXPORT_OK = qw(git_rev_parse checkout_git_repo_and_branch
+  checkout_git_refspec handle_generated_assets load_test_schedule);
+
+sub git_rev_parse ($dirname) {
+    my $version = (-d "$dirname/.git" ? qx{(git config --global --get safe.directory | grep -q $dirname || git config --global --add safe.directory $dirname) && git -C $dirname rev-parse HEAD} : '') || 'UNKNOWN';
```

Comment:
`.git` can also be just a file and not a directory.

Reply:
ok, added a commit